src/components/App/App.js

```diff
@@ -1,62 +1,36 @@
 import React, { lazy, Suspense } from 'react';
 import { IntlProvider } from 'react-intl';
 import { library } from '@fortawesome/fontawesome-svg-core';
-import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import { BrowserRouter } from 'react-router-dom';
 import flatten from 'flat';
 
 import icons from 'assets/icons';
-import AppLocale from 'locales';
 import ErrorBoundary from 'components/ErrorBoundary';
 import Loading from 'components/Loading';
-import ProtectedRoute from 'components/ProtectedRoute';
+import { useAuthentication } from 'hooks/auth';
 import { useLocale } from 'hooks/locale';
+import AppLocale from 'locales';
 
-import styles from './App.module.scss';
+import './App.module.scss';
 
-const ForgotPasswordPage = lazy(() => import('pages/ForgotPasswordPage'));
-const HomePage = lazy(() => import('pages/HomePage'));
-const NoMatchPage = lazy(() => import('pages/NoMatchPage'));
-const SettingsPage = lazy(() => import('pages/SettingsPage'));
-const SignInPage = lazy(() => import('pages/SignInPage'));
-const SignUpPage = lazy(() => import('pages/SignUpPage'));
+const AuthenticatedApp = lazy(() =>
```

Comment:
I don't like it either, it gives me the wrong idea. Maybe something like `AuthenticatedUsersApp`? It's borderline too long, but at least to me, it gives a more accurate idea. What do you think?

Reply:
What about AuthenticatedUser? AuthenticatedUsersApp feel to verbose